TheMET/TheMET/ViewControllers/CategoryViewController.swift

```diff
@@ -91,15 +91,59 @@ class CategoryViewController: UIViewController, UITableViewDelegate, UITableView
     
     private func reloadCategory() {
         self.contentStatus = .loading
-        self.loadListOfArts { [weak self] arts in
-            guard let arts = arts else {
+        self.loadArtCellDataList()
+    }
+    
+    private func loadArtCellDataList() {
+        guard let id = self.departmentId else {
+            self.contentStatus = .failed
+            return
+        }
+        self.metAPI.objects(departmentIds: [id]) { [weak self] objectsResponse in
+            guard let objectsResponse = objectsResponse else {
                 self?.contentStatus = .failed
                 return
             }
-            self?.contentStatus = .loaded(arts)
+            var artCellDataList: [ArtCellData] = []
+            for artId in objectsResponse.objectIDs {
+                let artCellData = ArtCellData(artID: artId, artData: .placeholder)
+                artCellDataList.append(artCellData)
+            }
+            self?.contentStatus = .loaded(artCellDataList)
+            self?.reloadCategory(artIDs: objectsResponse.objectIDs, artCellDataList: artCellDataList)
+        }
+    }
+    
+    private func reloadCategory(artIDs: [ArtID], artCellDataList: [ArtCellData]?) {
```

Comment:
`artCellDataList` argument is not used that much in this method and can be mistaken for `artCellDataList` defined at line 126. Let's remove it at all. Also, it is always going to be non-nil as the `artCellDataList`(line 107) is non-optional.

Reply:
+